assets/javascripts/discourse/templates/connectors/group-reports-nav-item/nav-item.hbs

```diff
@@ -0,0 +1 @@
+{{ group-reports-nav-item group = this.group }}
```

Comment:
```suggestion
{{group-reports-nav-item group = this.group}}
```

Reply:
Does this work properly? You shouldn't need to use `this` in a hbs template.

Reply:
It does work, but I can certainly remove `this`.